services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```

Comment:
Both `Movie.objects.get(id=movie_id)` and `CinemaHall.objects.get(id=cinema_hall_id)` will raise a `DoesNotExist` exception if the specified movie or cinema hall does not exist. Consider handling these exceptions to provide a more user-friendly error message or behavior.